k8s-configuration/artauction-deployment.yml

```diff
@@ -67,5 +78,5 @@ spec:
     app: app
   ports:
     - protocol: TCP
-      port: 8080
-      targetPort: 8080
+      port: 8081
+      targetPort: 8081
```

Comment:
will this work with mongo-express?

Reply:
remove mongo express cause we are using compass